client/plots/matrix.interactivity.js

```diff
@@ -367,14 +367,13 @@ export function setInteractivity(self) {
 				const v = c.value
 				const p = v.pairlst
 				const dtLabel = v.origin ? `${v.origin} ${dt2label[v.dt]}` : dt2label[v.dt]
-				const label =
-					c.label == self.config.settings.hierCluster?.termGroupName
```

Comment:
v.value may not always be appropriate even if it's present, and it also would miss out on zero values which may be a valid v.value